src/data.js

```diff
@@ -178,99 +179,112 @@ export const steps = [
   },
   {
     id: 'new-rights',
-    title:'New rights',
-    items: [,
+    title: 'Novos direitos',
+    items: [
       {
         id: 1,
-        title: 'Your customers can easily request access to their personal information',
+        title: 'Seus clientes podem requisitar acesso aos seus dados pessoais de forma simples.',
         role: ['processor', 'controller'],
-        description: 'If you do not already have a process defined for this, we\'ve made an easy online form below.',
+        description: 'Se você ainda não possui um processo definido para isso, criamos um formulário on-line fácil abaixo.',
```

Comment:
Sugestão:
```suggestion
        description: 'Se você ainda não possui um processo definido para isso, criamos um fácil formulário on-line abaixo.',
```
Não tenho certeza, o que acha?
Mesma coisa para as outras ocorrências abaixo.